src/lib/isDesktop.ts

```diff
@@ -1,5 +1,8 @@
 // https://github.com/electron/electron/issues/2288#issuecomment-337858978
 // Thank you
 export function isDesktop(): boolean {
-  return navigator.userAgent.toLowerCase().indexOf('electron') > -1
+  return typeof navigator !== 'undefined' &&
```

Comment:
Gotcha,

When `helpRole.ts` calls `withSiteBaseURL` the `env()` object will call `isDesktop()`. This will fail if called in the electron runtime.

We need to check if `navigator` is present or not. If it is not present that means you are in desktop. If is present then do the userAgent check.